### ISSUE TITLE
Background Color for Nav Menu Normal Tab

### ISSUE THREAD

**Issue by sudhirsahni** [elementor/elementor#10301]
Hello,

I am using Elementor Pro and using it's lot of features and really like Elementor effort... it's really a great Tools.

Currently I working on a client website and try to use Nav Menu Normal Tab Background and there not this option available like Hover or Active Tab, I don't want to add background i whole Nav, Just for Like Active or Hover Option working..

## Prerequisites

- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by vgstef** [comment]
Yeah, I'm facing the same problem! How come is there no background color for the normal menu item?
